src/modules/peek/Peek.UI/Extensions/IShellItemExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static IFileSystemItem ToIFileSystemItem(this IShellItem shellItem)
 
         private static string GetPath(this IShellItem shellItem)
         {
-            string path = string.Empty;
+            string path;
```

Comment:
Default value is null which can be problematic, why change it?

Reply:
I'm thinking the try and catch code will prevent problem?